packages/insomnia-importers/README.md

```diff
@@ -17,13 +17,13 @@ Insomnia v3 format.
 For usage on **command line**, install globally
 
 ```bash
-npm install -g insomnia-importers
+yar add --global insomnia-importers
```

Comment:
yarn reference here

Reply:
arg.  I thought I caught them all.  must have been a typo.

Reply:
resolved in https://github.com/dimitropoulos/insomnia/pull/4/commits/552d5ec20193f79b013914cc3c542c8d686c7fdd